src/components/patientsList/item/Item.js

```diff
@@ -12,34 +12,30 @@ import {
 } from "../../../constants/styles/typography";
 import PatientBasicData from "../../../constants/propTypes/basicDataPropTypes";
 
-const Item = ({ onPress, item: patient, data: patientBasicData }) => {
+const Item = ({ onPress, patient, patientsBasicData }) => {
   return (
     <View style={styles.item}>
       <TouchableOpacity onPress={onPress}>
         <Header patient={patient} />
         <InterviewInfo
           icon="diagnosis"
           name="Diagnoza"
-          value={
-            patient.diagnosis ? patient.diagnosis.join(", ") : patient.code
-          }
+          value={patient.diagnosis ? patient.diagnosis.join(", ") : ""}
         />
         <InterviewInfo
           icon="medicines"
           name="Leki"
           value={
-            patientBasicData.medications_used
-              ? patientBasicData.medications_used
-              : ""
+            patientsBasicData.medications ? patientsBasicData.medications : ""
           }
         />
         <InterviewInfo
           icon="hospital"
           name="Liczba hospitalizacji"
           value={
-            patientBasicData.hospitalization_times
-              ? patientBasicData.hospitalization_times.toString()
-              : "0"
+            patientsBasicData.hospitalization_times
+              ? `${patientsBasicData.hospitalization_times}`
+              : ""
```

Comment:
w formularzu BasicDataScreen wartość jest zapisywana jako Int, czy to nie będzie problem?